src/main/java/org/gridsuite/shortcircuit/server/repositories/ShortCircuitAnalysisResultRepository.java

```diff
@@ -98,6 +99,16 @@ public Optional<ShortCircuitAnalysisResultEntity> find(UUID resultUuid) {
         return resultRepository.findByResultUuid(resultUuid);
     }
 
+    @Transactional(readOnly = true)
+    public Optional<ShortCircuitAnalysisResultEntity> findFullResults(UUID resultUuid) {
```

Comment:
Maybe you could add a comment to explain this code that is quite touchy or add a link to an article about it

Reply:
done